File: Host/app/src/main/java/com/demokiller/host/Person.java
package com.demokiller.host;

import com.demokiller.host.utils.PatchProxy;
import com.demokiller.robustpatch.ChangeQuickRedirect;

public class Person {
    public static ChangeQuickRedirect changeQuickRedirect;

    public static String toStr(String str) {
        if (changeQuickRedirect != null) {
            if (PatchProxy.isSupport(new Object[]{str}, null, changeQuickRedirect, true)) {
                return (String) PatchProxy.accessDispatch(new Object[]{str}, null, changeQuickRedirect, true);
            }
        }
        return "aaa";
    }

}

File: Host/app/src/main/java/com/demokiller/host/Person.kt
package com.demokiller.host

import com.demokiller.host.utils.PatchProxy
import com.demokiller.robustpatch.ChangeQuickRedirect

object Person {
    var changeQuickRedirect: ChangeQuickRedirect? = null
    fun toStr(str: String): String {
        if (changeQuickRedirect != null) {
            if (PatchProxy.isSupport(arrayOf<Any>(str), null, changeQuickRedirect, true)) {
                return PatchProxy.accessDispatch(arrayOf<Any>(str), null, changeQuickRedirect, true) as String
            }
        }
        return "aaa"
    }
}